### PR TITLE
Simplified datamask for ORB product

### PR DESCRIPTION
**Pull request by maawoo**
This implements a simplified `land` / `ocean water` datamask layer for the ORB product: 

image

### CODE REVIEW

File: S1_NRB/nrb.py [hunk 1]
```diff
@@ -748,10 +748,10 @@ def calc_product_start_stop(src_ids, extent, epsg):
 
 
 def create_data_mask(outname, datasets, extent, epsg, driver, creation_opt,
-                     overviews, overview_resampling, dst_nodata, wbm=None):
+                     overviews, overview_resampling, dst_nodata, wbm=None, orb=False):
     """
```
Comment (johntruckenbrodt) [inline S1_NRB/nrb.py:752]: I would prefer a different parameter than `orb`. It somehow implies that NRB is the default and ORB the exception whereas both are equally viable options. How about `product_type` with two options `nrb` and `orb`?
Furthermore, the parameter meeds a docstring description.

**Review by johntruckenbrodt — CHANGES_REQUESTED** [review]
One request..

File: S1_NRB/nrb.py (hunk 1 above)
Comment (maawoo) [reply S1_NRB/nrb.py:752]: Sounds good. I've implemented it according to https://github.com/SAR-ARD/S1_NRB/commit/45056e9882a4f4da0185f32e362882c5241123ca but it makes sense to just leave the top-level hidden (i.e., `mode` in config) and already properly implement the option into functions. 

I would need to do the same in #121 with `nrb.format` and `metadata.extract.meta_dict`. The base branch of this PR is `orb/accommodate` so I could merge and then handle this as a single commit in #121. Is that okay or better to keep them separate?

Comment (johntruckenbrodt) [reply S1_NRB/nrb.py:752]: Great. Your approach sounds good, it's better to have the change in a single commit.

**Pull request merged**